lnrpc/lightning.proto

```diff
@@ -4402,8 +4412,9 @@ message PolicyUpdateRequest {
     // If true, min_htlc_msat is applied.
     bool min_htlc_msat_specified = 8;
 
-    int32 inbound_base_fee_msat = 10;
-    int32 inbound_fee_rate_ppm = 11;
+    // Optional inbound fee. If unset, the previously set value will be
+    // retained [EXPERIMENTAL].
```

Comment:
A bit confused about this flag `EXPERIMENTAL`, when will this stop being experimental?

Reply:
I added this because the whole inbound fee implentation is mentioned as experimental in the release notes. For other new features I have also seen the flag in the proto files.
But I can remove it  if it is rather confusing.

Reply:
I think fine to retain the flag. Indicates that the API may change in the near future.